Codecademy/gamut
# feat(Illustration): Add OfficeAnnouncement Illustration

### Overview



Adds the OfficeAnnouncement illustration.



<img width="300" alt="Screenshot 2024-04-12 at 5 06 42 PM" src="https://github.com/Codecademy/gamut/assets/14988121/473adec5-bd16-4537-a4bf-f8306cd39404">

### PR Checklist

- [x] Related to designs: https://www.figma.com/file/7Oe7eNRzW9J0KbShrfbiPy/Events?type=design&node-id=2288-19913&mode=design&t=B3O8IQiHEQUlS87t-4
- [x] Related to JIRA ticket: [LEGO-3797](https://codecademy.atlassian.net/browse/LEGO-3797)

#### Testing Instructions

Check the new illustration in the Storybook preview: https://6619bc47a254bc105779210a--gamut-preview.netlify.app/?path=/docs/atoms-illustrations--editable-illustration

Check the new illustration in the portal-app preview environment: https://tengu.codecademy.com/learn?page=communityEvents

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Portal       | [Portal Link](https://github.com/codecademy-engineering/mono/pull/5773)  | [Portal Env](https://tengu.codecademy.com/learn?PR_ENV=portal-app-pr-5773)   |




[LEGO-3797]: https://codecademy.atlassian.net/browse/LEGO-3797?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
